config.go

```diff
@@ -0,0 +1,280 @@
+package envbuilder
+
+import (
+	"github.com/coder/serpent"
+)
+
+type Config struct {
```

Comment:
Convention: It is a good practice to add comments to exposed structure fields (capital letter = exposed = visible outside the package).

See: https://github.com/coder/serpent/blob/00b6c3f9d4bced45b5f35278853c07f0548ba53f/command.go#L25